index.js

```diff
@@ -229,7 +234,7 @@ module.exports = {
                       }
                       const newTangleOpts = {
                         tangles: ['epoch'],
-                        isValid: () => true,
+                        isValid: isInit,
```

Comment:
as I said in the other PR, keen to see `isInitNewEpoch` or something to make sure this is really tight